bash_103_combining_commands/bash_103_combining_commands.md

```diff
@@ -0,0 +1,463 @@
+
+The "greater than" symbol, also known as an arrow or more specifically "right arrow" lets us redirect output. On a standard American qwerty keyboard, the `>` is located on the same key as the period `.`
+
+Try running the following code:
+
+```
+wc Animals.csv > animals.txt
+```
+
+You won't see any output because it was redirected to the newly created file `animals.txt`. Use `cat` or another program to check that the file `animals.txt` contains the text:
+
+```
+      19      35     341 Animals.csv
+```
+
+A single arrow will replace anything that was already in the file with the new content. If we don't want to replace the content in `animals.txt` we can append (add new lines after the existing lines) to the end of the file using double arrows `>>`:
+
+```
+sort Animals.csv >> animals.txt
+```
+
+Again there is no output shown because it was redirected into the file `animals.txt`. Check that `animals.txt` now contains the word count at the top, followed by a sorted copy of the data from `Animals.csv`.
+
+<div class = "learnmore">
+In the Bash language, right arrows redirect output, and left arrows (or "less than" sign) redirect input.
+</div>
+
+### Quiz: inputs and outputs
+
+Suppose you have acquired a new file with the uninformative name `newfile.txt`. You look at its contents and see that `cat newfile.txt` returns
+
+```
+Callinectes sapidus
+
+blue crab,crustacean
+
+Source: https://www.fisheries.noaa.gov/species/blue-crab
+
+The blue crab is a highly sought-after shellfish. Blue crabs live up and down the Atlantic Coast and in the Gulf of Mexico and are caught by both commercial and recreational fishermen. Its scientific name—Callinectes sapidus—translated from Latin means 'beautiful savory swimmer.'
+
+Blue crabs are the most valuable fishery in the Chesapeake Bay. They are also major predators of benthic communities and are prey for many other fish species. Blue crabs are so treasured in the region that the blue crab is the Maryland state crustacean.
+
+Length: 9 in
+Weight: .33 lbs
+```
+
+You want to convert information in this file into a similar format to your other animal files, while avoiding typing the information in by hand.
+
+1. What command would you type to create a new file `blue_crab.txt` that contains only the first line of this file?
+
+[[head -1 newfile.txt > blue_crab.txt]]
+<script>
+  let input = "@input".trim().toLowerCase();
+  input == "head -1 newfile.txt > blue_crab.txt" || input == "head -1 newfile.txt >> blue_crab.txt";
+</script>
+***
+<div class = "answer">
+The output of `head -1 newfile.txt` is the first line of `newfile.txt`. Since no file named  `blue_crab.txt` previously existed, overwriting (`>`) and appending (`>>`) that output have the same result.
+</div>
+***
+
+2. What command would you type to create a new file `blue_crab.dat` that contains only the last two lines of this file?
```

Comment:
This question doesn't show up unless you get the correct answer for the first question.

Reply:
I think this connected to the newlines.